### ISSUE TITLE
Suggestion: Display Name in, /town

### ISSUE THREAD

**Issue by Mitsu13Ion** [TownyAdvanced/Towny#3576]
Hello, i can use the displayname in /town ?
because my server work on roleplay name

**Comment by LlmDl** [comment]
No, but you could be using the Town or Nation titles and surnames which I believe are used in the /t, /n and /res screens.

**Comment by Mitsu13Ion** [comment]
And you could add displayname in config ?

> 
> 
> No, but you could be using the Town or Nation titles and surnames which I believe are used in the /t, /n and /res screens.
